### ISSUE TITLE
Website not working

### ISSUE THREAD

**Issue by IZeeLikeEyeZee** [capjamesg/bsky.link#28]
when I visit bsky.link I get this:

> Welcome to nginx!
> If you see this page, the nginx web server is successfully installed and working. Further configuration is required.
> 
> For online documentation and support please refer to [nginx.org](http://nginx.org/).
> Commercial support is available at [nginx.com](http://nginx.com/).
> 
> Thank you for using nginx.
>

**Comment by someone83474** [comment]
Yep, sadly for a while now. I build something similar (still beta), maybe you want to give it a try https://embed-bsky.link (yeah, the cool domain was taken :).

**Comment by capjamesg** [comment]
I don't know why I didn't respond to this earlier. My apologies!

The URL embedding tool is still active: https://bsky.link/?url=https://bsky.app/profile/jamesg.blog/post/3jwbah3ssi32y

The syntax is:

```
https://bsky.link?url=[your bluesky post]
```

Also, cool project @someone83474! bsky.link used to support embedding profiles but after problematic usage of this feature, we removed it. We haven't removed it from the home page yet, though, so the home page is down while we do.

**Comment by Calipso5U** [comment]
> Yep, sadly fro a while now. I build something similar (still beta), maybe you want to give it a try https://embed-bsky.link (yeah, so cool domain was taken :).

Awesome project, @someone83474 ! I love the look of your version. Is there any chance you're going to include profile embedding in the future? I've been looking for a way to embed my project's profile on my website and your build is just perfect.

**Comment by someone83474** [comment]
> Awesome project, @someone83474 ! I love the look of your version. Is there any chance you're going to include profile embedding in the future? I've been looking for a way to embed my project's profile on my website and your build is just perfect.

Thanks :) - Yeah, I let me see if I can add that feature.

btw. it is now https://bluesky.lol (easier to remember, old links still work).

**Comment by someone83474** [comment]
> Also, cool project @someone83474! bsky.link used to support embedding profiles but after problematic usage of this feature, we removed it. We haven't removed it from the home page yet, though, so the home page is down while we do.

Thanks :) - What was the reason you removed profile embedding?

**Comment by Neustradamus** [comment]
The link in "About" section and in README does not work.
